bin/proof-generator/include/nil/proof-generator/file_operations.hpp

```diff
@@ -101,27 +98,24 @@ namespace nil {
             return file;
         }
 
-        auto open_file_r(const std::string& path, std::ios_base::openmode mode = std::ios_base::in) {
-            return open_file<std::ifstream>(path, mode);
+        auto open_file_r(const std::string& path, std::ios_base::openmode extra_mode = std::ios_base::in) {
```

Comment:
Do we even need these functions? Maybe copy its body on the caller side, it will be more readable?

Reply:
Yeah, a bit redundant. Removed it